src/components/Form.tsx

```diff
@@ -84,3 +101,31 @@ export const FormHint: FunctionComponent<FormHintProps> = ({
     {children}
   </div>
 );
+
+export const Input: FunctionComponent<InputProps> = ({
+  name,
+  type = "text",
+  spellCheck = true,
+}: InputProps): JSX.Element => (
+  <input
+    className="govuk-input"
+    name={name}
+    type={type}
+    spellCheck={spellCheck}
+  />
+);
+
+export const Select: FunctionComponent<SelectProps> = ({
+  name,
+  defaultValue,
+  children,
+}: SelectProps): JSX.Element => (
+  <select className="govuk-select" name={name} defaultValue={defaultValue}>
+    {children}
+  </select>
+);
```

Comment:
https://design-system.service.gov.uk/components/select/

![image](https://user-images.githubusercontent.com/54271433/107084867-8b7cf900-67ef-11eb-99d9-9ae7e432d82f.png)

Reply:
You're spot on, my bad.  `id` added to `Select`